firebase-common/src/main/java/com/google/firebase/concurrent/FirebaseExecutors.java

```diff
@@ -33,6 +35,91 @@ public static Executor newSequentialExecutor(Executor delegate) {
     return new SequentialExecutor(delegate);
   }
 
+  /**
+   * Returns a {@link Executor} that limits the number of running tasks at a given time.
+   *
+   * <p>The executor uses delegate in order to {@link Executor#execute(Runnable) execute} each task,
```

Comment:
```suggestion
   * <p>The executor uses the {@code delegate} in order to {@link Executor#execute(Runnable) execute} each task,
```

Similar to newPausableExecutor docs

Reply:
here and below